Kickstarter-iOS/DataSources/CategorySelectionDataSource.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+import KsApi
+import Library
+import UIKit
+
+internal final class CategorySelectionDataSource: ValueCellDataSource {
+  private var categorySectionTitles: [String] = []
```

Comment:
I've stored the section titles in here for now. I think ideally we'd want to extend the functionality of our `ValueCellDataSource` object to manage section header view data sources as well, but I felt that this was out of the scope of this MVP.